.circleci/config.yml

```diff
@@ -11,10 +11,12 @@ jobs:
       - image: circleci/python:3.6.1
         environment:
           FLASK_CONFIG: testing
+          DATABASE_URL: postgresql://root@localhost/circle_test
+          HOOK_SECRET_KEY: SECRETS
+      - image: circleci/postgres:10.4-alpine-ram
```

Comment:
That's a good point. I didn't want to leave the comment where it was because I thought it made the block harder to read. But, yes, I can make that clearer!